setup_deb.sh

```diff
@@ -68,133 +68,142 @@ log "Installing python dependencies ..."
-   multiprocess==0.70.14 \
-   pyarrow==12.0.1 \
-   dill==0.3.6 \
-   aiohttp==3.8.4 \
-   multidict==6.0.4 \
-   yarl==1.9.2 \
-   async_timeout==4.0.2 \
-   aiosignal==1.3.1 \
-   frozenlist==1.3.3 \
-   xxhash==3.2.0 \
-   lazy_loader==0.2 \
-   audioread==3.0.0 \
-   soxr==0.3.5 \
-   msgpack==1.0.5 \
-   jiwer==3.0.2 \
-   click==8.1.3 \
-   rapidfuzz==2.13.7 \
-   llvmlite==0.42.0 \
-   decorator==5.1.1 \
-   fsspec==2023.6.0 \
-   unicode==2.9 \
-   unidecode==1.3.6 \
-   inflect==6.0.4 \
-   pydantic==1.10.9 \
-   cffi==1.15.1 \
-   attrs==23.1.0 \
-   albumentations==0.4.3 \
-   pudb==2019.2 \
-   imageio==2.9.0 \
-   imageio-ffmpeg==0.4.2 \
-   pytorch-lightning==1.9.1 \
-   ftfy==6.1.1 \
-   imwatermark==0.0.2 \
-   PyWavelets==1.4.1 \
-   torchmetrics==0.6 \
-   omegaconf==2.3.0 \
-   lightning_utilities==0.9.0 \
-   wcwidth==0.2.6 \
-   antlr4-python3-runtime==4.9.3 \
-   test-tube>=0.7.5 \
-   streamlit>=0.73.1 \
-   einops==0.3.0 \
-   webdataset==0.2.5 \
-   open-clip-torch==2.7.0 \
-   gradio==3.13.2 \
-   kornia==0.6 \
-   invisible-watermark>=0.1.5 \
-   streamlit-drawable-canvas==0.8.0 \
-   safetensors>=0.3.1
+    cycler==0.11.0 \
+    filelock==3.6.0 \
+    future==0.18.2 \
+    huggingface-hub==0.20.2 \
+    joblib==1.1.0 \
+    kiwisolver==1.4.2 \
+    matplotlib==3.5.1 \
+    nnunet==1.7.1 \
+    packaging==21.3 \
+    Pillow==9.1.0 \
+    pyparsing==3.0.8 \
+    python-dateutil==2.8.2 \
+    pytz==2022.1 \
+    pyyaml==6.0 \
+    regex==2022.3.15 \
+    sacremoses==0.0.49 \
+    scikit-image==0.19.2 \
+    scikit-learn==1.0.2 \
+    threadpoolctl==3.1.0 \
+    tokenizers==0.19.1 \
+    tabulate==0.9.0 \
+    regex==2022.3.15 \
+    portalocker==2.6.0 \
+    lxml==4.9.2 \
+    colorama==0.4.6 \
+    thop \
+    psutil==5.9.5 \
+    multiprocess==0.70.14 \
+    pyarrow==12.0.1 \
+    dill==0.3.6 \
+    aiohttp==3.8.4 \
+    multidict==6.0.4 \
+    yarl==1.9.2 \
+    async_timeout==4.0.2 \
+    aiosignal==1.3.1 \
+    frozenlist==1.3.3 \
+    xxhash==3.2.0 \
+    lazy_loader==0.2 \
+    audioread==3.0.0 \
+    soxr==0.3.5 \
+    msgpack==1.0.5 \
+    jiwer==3.0.2 \
+    click==8.1.3 \
+    rapidfuzz==2.13.7 \
+    llvmlite==0.42.0 \
+    decorator==5.1.1 \
+    fsspec==2023.6.0 \
+    unicode==2.9 \
+    unidecode==1.3.6 \
+    inflect==6.0.4 \
+    pydantic==1.10.9 \
+    cffi==1.15.1 \
+    attrs==23.1.0 \
+    albumentations==0.4.3 \
+    pudb==2019.2 \
+    imageio==2.9.0 \
+    imageio-ffmpeg==0.4.2 \
+    pytorch-lightning==1.9.1 \
+    ftfy==6.1.1 \
+    imwatermark==0.0.2 \
+    PyWavelets==1.4.1 \
+    torchmetrics==0.6 \
+    omegaconf==2.3.0 \
+    lightning_utilities==0.9.0 \
+    wcwidth==0.2.6 \
+    antlr4-python3-runtime==4.9.3 \
+    test-tube \
+    streamlit \
+    einops==0.3.0 \
+    webdataset==0.2.5 \
+    open-clip-torch==2.7.0 \
+    gradio==3.13.2 \
+    kornia==0.6 \
+    invisible-watermark \
+    streamlit-drawable-canvas==0.8.0 \
+    safetensors >=0.1.1 >=0.7.5 >=0.73.1 >=0.1.5 >=0.3.1
```

Comment:
Looks like version specifiers of a few different packages got moved into this line

Reply:
Good catch, missed that.